### PR TITLE
Fix SickRage/sickrage-issues/issues/693

### PR DESCRIPTION
**Pull request by fernandog**
https://github.com/SickRage/sickrage-issues/issues/693

### CODE REVIEW

**Comment by SickRage2** [comment]
Thanks for the pull request! Before a real human comes by, please make sure your PR has all of the below criteria checked

- [x] Give a description on what the PR is for.
- [x] Make sure your PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review

Please make sure you also read [contribution guide](https://github.com/SickRage/SickRage/blob/master/contributing.md) and followed all of the steps.

Thanks!

File: sickbeard/providers/transmitthenet.py [hunk 1]
```diff
@@ -137,14 +139,14 @@ def search(self, search_strings, age=0, ep_obj=None):  # pylint: disable=too-man
...
-                                title = title.replace("[", "").replace("]", "").replace("/ ", "")
+                                title = title.replace("[", "").replace("]", "").replace("/ ", "") if title else ''
 
                             torrent_size = temp_anchor['data-filesize']
                             size = convert_size(torrent_size) or -1
 
                             temp_anchor = torrent_row.find('span', class_='time').parent.find_next_sibling()
-                            seeders = try_int(temp_anchor.text.strip())
-                            leechers = try_int(temp_anchor.find_next_sibling().text.strip())
+                            seeders = try_int(temp_anchor.text.strip(), 1)
```
Comment (miigotu) [inline sickbeard/providers/transmitthenet.py]: remove these 2 changes.

File: sickbeard/providers/transmitthenet.py [hunk 2]
```diff
@@ -137,14 +139,14 @@ def search(self, search_strings, age=0, ep_obj=None):  # pylint: disable=too-man
...
+                            leechers = try_int(temp_anchor.find_next_sibling().text.strip(), 0)
```
Comment (miigotu) [inline sickbeard/providers/transmitthenet.py]: ^

**Comment by fernandog** [comment]
@miigotu done

**Pull request merged**